tests/metrics/genai/prompts/test_v1.py

```diff
@@ -84,22 +94,38 @@ def test_evaluation_model_output():
         - Score 5: the answer correctly answer the question and not missing any major aspect
 
     Examples:
-        Input: This is an input
-        Provided output: This is an output
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is a justification
-
-        Input: This is an example input 2
-        Provided output: This is an example output 2
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is an example justification 2
-
-    And you'll need to submit your grading for the correctness of the output,
-    using the following in json format:
-    Score: [your score number for the correctness of the output]
-    Justification: [your step by step reasoning about the correctness of the output]
+        Input:
+        This is an input
+
+        Output:
+        This is an output
+
+        Additional information used by the model:
+        key:
+        ground_truth
+        value:
+        This is an output
+
+        score: 4
+        justification: This is a justification
+
+
+        Input:
+        This is an example input 2
+
+        Output:
+        This is an example output 2
+
+        Additional information used by the model:
+        key: ground_truth
+        value: This is an output
+
+        score: 4
+        justification: This is an example justification 2
```

Comment:
These examples are incorrect because the output is not in JSON format

Reply:
Yes because our prompt template parsing logic builds variables using `{}` as a way to find the variables we need to fill in the values.
So if I convert the examples response into JSON format it takes them as keys as well that we need to add values for. So it might not be possible to do this :(